subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -5432,6 +5478,21 @@ static inline int ctrl_rx(memq_link_t *link, struct node_rx_pdu **rx,
 			goto ull_conn_rx_unknown_rsp_send;
 		}
 
+#if defined(CONFIG_BT_CTLR_PHY)
+		/* LL_ENC_REQ was received while LLCP_PHY_UPD was running */
+		if (conn->llcp_type == LLCP_PHY_UPD &&
+		    conn->llcp_ack != conn->llcp_req) {
+			 /* Adjust ack due to decrement below, to prevent
+			  * failures
+			  */
+			conn->llcp_ack += 2U;
+			/* Remember that LLCP_PHY_UPD was interrupted */
+			conn->llcp.encryption.pend_llcp_type = LLCP_PHY_UPD;
+		} else {
+			conn->llcp.encryption.pend_llcp_type = LLCP_NONE;
+		}
```

Comment:
```suggestion
		/* LL_ENC_REQ was received while local procedure is running */
		nack = ((conn->llcp_req - conn->llcp_ack) & 0x03 == 0x02);
		if (unlikely(nack)) {
			if (conn->llcp_type == LLCP_PHY_UPD) {
				nack = 0U;
				
				/* Adjust ack due to decrement below, to prevent
				 * failures
				 */
				conn->llcp_ack += 2U;
				
				/* Remember that LLCP_PHY_UPD was interrupted */
				conn->llcp.encryption.pend_llcp_type = LLCP_PHY_UPD;
			} else {
				break;
			}
		}
```